apps/docs/components/reference/RefFunctionSection.tsx

```diff
@@ -114,7 +114,13 @@ const RefFunctionSection: React.FC<IRefFunctionSection> = (props) => {
                 >
                   {item.examples &&
                     item.examples.map((example, exampleIndex) => {
-                      const exampleString = ''
+                      const exampleString =
+                        '' +
+                        (example.code &&
+                          example.code
+                            .trim()
+                            .replace(/^```(js|ts|dart|c#|kotlin)/, '')
```

Comment:
Thoughts on making this more generic so that we don't need to continually add new languages? Maybe `.*` (needs to be tested)?

Reply:
I think something like that makes sense. I’ll try and test it out later today.

Reply:
Changed to the `.*` suggestion.  This will now correctly replace code examples that have 3 backticks with a language (\`\`\`js) as well as 3 backticks without a language (\`\`\`).